### PR TITLE
ARVIR - Animates the line into position

### PR DESCRIPTION
**Pull request by orta**
Fixes EV-165
#trivial

### CODE REVIEW

File: Artsy/View_Controllers/ARVIR/AR/ARVIRHorizontalPlaneInteractionController.m [hunk 1]
```diff
@@ -288,6 +291,23 @@ - (void)renderWhenPlacingWall:(ARFrame *)frame API_AVAILABLE(ios(11.0));
                 ghostWall.constraints = @[lookAtCamera];
 
                 self.ghostWallLine = ghostWall;
+
+                // Handle a one-off animation
+                if(!self.hasShownGhostWallLineOnce) {
+                    self.hasShownGhostWallLineOnce = true;
```
Comment (alloy) [inline Artsy/View_Controllers/ARVIR/AR/ARVIRHorizontalPlaneInteractionController.m]: Note to self, need to reset this too after rebasing.

**Comment by alloy** [comment]
👍

File: Artsy/View_Controllers/ARVIR/AR/ARVIRHorizontalPlaneInteractionController.m (hunk 1 above)
Comment (alloy) [reply Artsy/View_Controllers/ARVIR/AR/ARVIRHorizontalPlaneInteractionController.m]: Oh, btw, this should be using `YES` :)

Comment (orta) [reply Artsy/View_Controllers/ARVIR/AR/ARVIRHorizontalPlaneInteractionController.m]: Hah, man, JS has ruined me

Comment (alloy) [reply Artsy/View_Controllers/ARVIR/AR/ARVIRHorizontalPlaneInteractionController.m]: lol

**Pull request merged**